source/extensions/filters/network/dubbo_proxy/BUILD

```diff
@@ -161,3 +161,19 @@ envoy_cc_library(
         "//include/envoy/stats:stats_macros",
     ],
 )
+
+envoy_cc_library(
+    name = "app_exception_lib",
+    srcs = ["app_exception.cc"],
+    hdrs = ["app_exception.h"],
+    deps = [
+        ":deserializer_interface",
+        ":message_lib",
+        ":metadata_lib",
+        ":protocol_interface",
+        "//include/envoy/buffer:buffer_interface",
+        "//source/common/buffer:buffer_lib",
+        "//source/common/common:byte_order_lib",
+        "//source/extensions/filters/network/dubbo_proxy/filters:filter_interface",
```

Comment:
Remove?

Reply:
Filter_interface dependency is needed, because AppException inherited from DubboFilters: :DirectResponse。